src/commands/info.rs

```diff
@@ -0,0 +1,68 @@
+use std::env;
+use std::path::{PathBuf, MAIN_SEPARATOR};
+
+use prettytable::{Table, Row, Cell};
+
+use crate::options::Options;
+use crate::platform;
+use crate::table;
+
+
+fn dir_to_str(path: PathBuf) -> String {
+    let mut rv = path.display().to_string();
+    rv.push(MAIN_SEPARATOR);
+    rv
+}
+
+pub async fn info(_options: &Options)-> Result<(), anyhow::Error>
+{
+    let mut table = Table::new();
+
+    table.add_row(Row::new(vec![
+        Cell::new("Cache"),
+        Cell::new(&dir_to_str(platform::cache_dir()?)),
+    ]));
+    table.add_row(Row::new(vec![
+        Cell::new("Config"),
+        Cell::new(&dir_to_str(platform::config_dir()?)),
+    ]));
+    if let Ok(current_exe) = env::current_exe() {
+        table.add_row(Row::new(vec![
+            if current_exe == platform::binary_path()? {
+                Cell::new("CLI Binary")
+            } else {
+                Cell::new("Custom Binary")
+            },
+            Cell::new(&current_exe.display().to_string()),
```

Comment:
I think we should put installation target too. So you can get `./edgedb cli info` and know installation path?

Reply:
I'm sorry I'm not getting it - "installation target" as in `platform::binary_path()` or the installed *versions* of EdgeDB servers?

Reply:
`./edgedb cli install` is already saying:

```
The edgedb binary will be placed in the user bin directory located at:
/Users/fantix/Library/Application Support/edgedb/bin
```

It's a bit redundant to have `./edgedb cli info` I think?

Reply:
In that sense displaying `current_exe()` makes even smaller sense because you can run `which edgedb`.

Reply:
OK I think you have two proposals here:

1. Add `edgedb cli info` to show `platform::binary_path()` regardless of its existance - I don't think we need this because `./edgedb cli install` will show that path anyways.
2. Drop `Custom Binary` in `edgedb info` and always show `platform::binary_path()` as `Binary` regardless of its existance.

If my understanding is correct, I'll fix (2) in a following PR.